content/doc/developer/publishing/source-code-hosting.adoc

```diff
@@ -9,3 +9,42 @@ The canonical source code repository is in the +jenkinsci+ GitHub organization t
 Some plugins we distribute are currently maintained outside the +jenkinsci+ GitHub organization for historical reasons, but this is discouraged.
 
 Plugin maintainers typically have repository admin permissions for plugins they maintain, but some features (like transfer and deletion of repos) are disabled.
+
+This table shows the permissions currently granted to contributors in the +jenkinsci+ GitHub repositories:
```

Comment:
Use backticks instead of plusses.

Reply:
Plusses and backticks are about evenly distributed in content/doc/developer, so we're not getting anywhere without an organized cleanup. Doing it in this one instance isn't worth the effort.